Description: fix lintian autotools-pkg-config-macro-not-cross-compilation-safe
Author: Roberto Lumbreras <rover@debian.org>

Index: dmtx-utils-0.7.6/configure.ac
===================================================================
--- a/configure.ac	2018-08-08 19:49:08.880725473 +0200
+++ b/configure.ac	2018-08-08 19:49:08.872725433 +0200
@@ -6,7 +6,7 @@
 AC_CONFIG_HEADERS([config.h])
 AC_CONFIG_FILES([Makefile])
 
-AC_PATH_PROG([PKG_CONFIG], [pkg-config], [no])
+PKG_PROG_PKG_CONFIG([PKG_CONFIG], [pkg-config], [no])
 
 AC_PROG_CC
 AC_PROG_LIBTOOL

